src/firmware_new/boards/frankie_v1/FreeRTOSConfig.h

```diff
@@ -64,7 +64,7 @@ void xPortSysTickHandler(void);
 #define configTICK_RATE_HZ ((TickType_t)1000)
 #define configMAX_PRIORITIES (56)
 #define configMINIMAL_STACK_SIZE ((uint16_t)1024)
-#define configTOTAL_HEAP_SIZE ((size_t)30720)
+#define configTOTAL_HEAP_SIZE ((size_t)80720)
```

Comment:
why this size in particular?

Reply:
I just "downloaded" 50kb more ram, nothing more
once we finalize a bit more things, we can justify with https://github.com/UBC-Thunderbots/Software/issues/1192